src/test/java/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -2542,39 +2542,60 @@ public void createCustomActivityEventAccountNotFound() throws Exception {
     public void getParticipantRosterNullPassword() throws JsonProcessingException {
         ParticipantRosterRequest request = new ParticipantRosterRequest.Builder().withPassword(null).build();
 
-        participantService.getParticipantRoster(TEST_APP_ID, TEST_USER_ID, request);
+        participantService.requestParticipantRoster(APP, TEST_USER_ID, request);
     }
 
     @Test(expectedExceptions = InvalidEntityException.class)
     public void getParticipantRosterBlankPassword() throws JsonProcessingException {
         ParticipantRosterRequest request = new ParticipantRosterRequest.Builder().withPassword("").build();
 
-        participantService.getParticipantRoster(TEST_APP_ID, TEST_USER_ID, request);
+        participantService.requestParticipantRoster(APP, TEST_USER_ID, request);
     }
 
     @Test(expectedExceptions = InvalidEntityException.class)
     public void getParticipantRosterInvalidPassword() throws JsonProcessingException {
         ParticipantRosterRequest request = new ParticipantRosterRequest.Builder().withPassword("badPassword").build();
 
-        participantService.getParticipantRoster(TEST_APP_ID, TEST_USER_ID, request);
+        participantService.requestParticipantRoster(APP, TEST_USER_ID, request);
     }
 
     @Test
     public void getParticipantRoster() throws JsonProcessingException {
+        account.setEmail(EMAIL);
+        account.setEmailVerified(TRUE);
+        when(accountService.getAccount(any())).thenReturn(Optional.of(account));
+        
         ParticipantRosterRequest request = new ParticipantRosterRequest.Builder().withPassword(PASSWORD).withStudyId(STUDY_ID).build();
 
         String queueUrl = "https://sqs.us-east-1.amazonaws.com/420786776710/Bridge-WorkerPlatform-Request-local";
         when(bridgeConfig.getProperty("workerPlatform.request.sqs.queue.url")).thenReturn(queueUrl);
 
         when(sqsClient.sendMessage(eq(queueUrl), anyString())).thenReturn(mock(SendMessageResult.class));
 
-        participantService.getParticipantRoster(TEST_APP_ID, TEST_USER_ID, request);
+        participantService.requestParticipantRoster(APP, TEST_USER_ID, request);
 
         String requestJson = "{\"service\":\"DownloadParticipantRosterWorker\",\"body\":{\"appId\":\"test-app\"," +
                 "\"userId\":\"userId\",\"password\":\"P@ssword1\",\"studyId\":\"studyId\"}}";
         verify(sqsClient).sendMessage(queueUrl, requestJson);
     }
 
+    @Test(expectedExceptions = BadRequestException.class)
+    public void getParticipantRoster_emailNotVerified() throws JsonProcessingException {
```

Comment:
This is nit-picky but could rename the tests from "get..." to "request..." to match the method rename.

Reply:
Updated all these tests with new method name. (not nit-picky!)